interfaces/cython/cantera/onedim.py

```diff
@@ -763,6 +770,21 @@ def check_blowoff(t):
         self.set_steady_callback(original_callback)
 
 
+class BurnerIonFlame(BurnerFlame, IonFlameBase):
+    """A burner-stabilized flat flame with ionized gas."""
+    __slots__ = ('burner', 'flame', 'outlet')
+
+    def __init__(self, gas, grid=None, width=None):
+        if not hasattr(self, 'flame'):
+            # Create flame domain if not already instantiated by a child class
+            self.flame = IonAxisymmetricStagnationFlow(gas, name='flame')
+
+        super(BurnerIonFlame, self).__init__(gas, grid, width)
+
+    def solve(self, loglevel=1, refine_grid=True, auto=False, stage=1, enable_energy=True):
+        IonFlameBase.solve(self, loglevel, refine_grid, auto, stage, enable_energy)
```

Comment:
If you list `IonFlameBase` as the first base class, then overriding this method will be unnecessary.